flink-table/flink-table-planner-blink/src/test/scala/org/apache/flink/table/planner/expressions/ScalarOperatorsTest.scala

```diff
@@ -62,6 +62,40 @@ class ScalarOperatorsTest extends ScalarOperatorsTestBase {
     )
   }
 
+  @Test
+  def testCompareOperator(): Unit= {
+
+    // f18 and f19 have same length.
+    testSqlApi(
+      "f18 > f19",
+      "true")
```

Comment:
We may also need to add some test in `SqlExpressionTest.testComparisonFunctions`